Snakefile

```diff
@@ -153,6 +155,8 @@ rule map_refseqids:
         uniprothits = output_dir / blastresults_dir / "{protid}.blasthits.uniprot.txt"
     benchmark:
         output_dir / benchmarks_dir / "{protid}.map_refseqids.txt"
+    conda:
+        "envs/map_refseqids.yml"
```

Comment:
`map_refseq_ids.yml`?

Reply:
That's probably better! I matched the rule name for the envs, which is `map_refseqids` for this rule.